backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -31,9 +33,6 @@ public Organization getOrganization() {
 
         List<DeviceType> currentDevices = _deviceService.fetchDeviceTypes();
         List<String> currentDeviceNames = currentDevices.stream().map(d->d.getName()).collect(Collectors.toList());
-        for (int i = 0; i < currentDevices.size(); i++) {
-            currentDeviceNames.add(currentDevices.get(i).getName());
-        }
         if (!currentDeviceNames.contains("Abbott IDNow")) {
             _deviceService.createDeviceType("Abbott IDNow", "ID Now", "Abbott");
```

Comment:
can this be called when the device already exists?

Reply:
Not safely.